config/examples/Anet/ET4-Pro/Configuration.h

```diff
@@ -2345,7 +2345,7 @@
 // 320x240, 3.2", FSMC Display From MKS
 // Normally used in MKS Robin Nano V1.2
 //
-#define MKS_ROBIN_TFT32
+//#define MKS_ROBIN_TFT32
```

Comment:
You should not add ANET_ET4_TFT28 or ANET_ET5_TFT35 too?

Reply:
It’s already enabled.